packages/opencensus-web-core/test/test-tracer.ts

```diff
@@ -17,10 +17,16 @@
 import * as webTypes from '@opencensus/web-types';
 import { Tracer } from '../src/trace/model/tracer';
 
+export interface WindowWithZone extends Window {
+  Zone?: Zone;
```

Comment:
I'm not sure the type here should be `Zone`, since that would be the type of an individual zone and not the type of the Zone class itself. I think you could use `Function`, which is vague but true.